File: dailyQuestion/2020-06/06-11/golang/solution_baoli.go
package main

import "fmt"

func main() {

	T := []int{73, 74, 75, 71, 69, 72, 76, 73}
	result := dailyTemperatures(T)
	fmt.Println(result)

}

func dailyTemperatures(T []int) []int {
	n := len(T)
	result := make([]int, n)
	if n == 1 {
		return result
	}
	for i := 0; i < n-1; i++ {
		for j := i + 1; j < n; j++ {
			if T[j] > T[i] {
				result[i] = j - i
				break
			}
		}
	}
	return result
}

File: dailyQuestion/2020-06/06-11/python/solution_baoli.py
# 暴力法
def dailyTemperatures(T: [int]) -> [int]:
    n = len(T)
    result = [0] * n
    for i in range(n-1) :
        for j in range(i+1,n) :
            if T[j] > T[i] :
                result[i] = j-i
                break
    return result

if __name__ == "__main__" :
    T = [73,74,75,71,69,72,76,73]
    result = dailyTemperatures(T)
    print(result)